python/pyspark/pandas/missing/series.py

```diff
@@ -59,7 +59,6 @@ class MissingPandasLikeSeries:
 
     # Properties we won't support.
     array = common.array(_unsupported_property)
-    duplicated = common.duplicated(_unsupported_property)
```

Comment:
There is no `duplicated` property as https://pandas.pydata.org/docs/reference/api/pandas.Series.html. It is a method actually.